Makefile

```diff
@@ -6,6 +6,9 @@ GIR_FILES = gir-files/Gtk-3.0.gir
 # Run `gir` generating the bindings
 gir : src/auto/mod.rs
 
+doc: $(GIR) $(GIR_FILES)
+	$(GIR) -m doc -c Gir.toml
```

Comment:
Don't want add "-o ../lgpl-docs\gtk\" ?

Reply:
I'd prefer not: we generate a `vendor.md`, then we can just grab it. We don't know for sure if the one starting the command has the `lgpl-docs` repository locally as well. Let's keep it simple. ;)